Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -57,6 +57,13 @@
 
 QPointer<MenuBar> MenuBar::s_menu_bar;
 
+const QString MenuBar::GetSignatureSelector() const
```

Comment:
```suggestion
QString MenuBar::GetSignatureSelector() const
```

const when returning by value doesn't do anything beneficial (and can even inhibit return value optimizations)